### ISSUE TITLE
Updating Twitter to X

### ISSUE THREAD

**Issue by arnavsharma2711** [monkeytypegame/monkeytype#4788]
### Did you clear cache before opening an issue?

- [X] I have cleared my cache

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Does the issue happen when logged in?

Yes

### Does the issue happen when logged out?

Yes

### Does the issue happen in incognito mode when logged in?

Yes

### Does the issue happen in incognito mode when logged out?

Yes

### Current Behavior

Show Twitter and its blue bird logo

### Expected Behavior

Show X and its new logo

### Steps To Reproduce

1. Home Screen

### Environment

- OS:
- Browser:
- Browser Version:

**Comment by skoom21** [comment]
would love to do this can someone assign this to me?

**Comment by Miodec** [comment]
> I have searched the existing issues

Duplicate of #4572. 

TLDR: I prefer the bird :)
